aws/letsencrypt.yaml

```diff
@@ -0,0 +1,39 @@
+# Cf. https://cert-manager.io/docs/configuration/acme/
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
```

Comment:
Seems like `Issuer`s should be namespaced (else you'd use a `ClusterIssuer`)?

Reply:
The ingress for curator service is defined in the default namespace which is also where the deployments live, the requirement is that Issuer must be in the same namespace as the ingress one so this is fine as I understand it.